recipes/galera.rb

```diff
@@ -110,7 +110,7 @@
   galera_cluster_nodes.each do |lnode|
     next unless lnode.name != node.name
     gcomm += ',' unless first
-    gcomm += lnode['fqdn']
+    gcomm += "#{lnode['fqdn']}:#{node['mariadb']['galera']['cluster_replication_port']}"
     first = false
```

Comment:
`node['mariadb']['galera']['cluster_replication_port']` should be `lnode['mariadb']['galera']['cluster_replication_port']` yes?  This way it uses the attribute in the node data found by the search and not what is defined locally.

The way it is now, if node1 has `['mariadb']['galera']['cluster_replication_port'] = 5678` and node2 has `['mariadb']['galera']['cluster_replication_port'] = 6789`, node1 will add `node2:5678` to the array which wouldn't work.